test/test_cuda/test_transformers.py

```diff
@@ -204,6 +206,45 @@ def test_mixed_bits(self):
             inputs = tokenizer(text, return_tensors="pt").to(model.device)
             tokenizer.decode(model.generate(**inputs, max_new_tokens=5)[0])
 
+    def test_mx(self):
```

Comment:
this file is only for testing same uts  in huggingface transformers, need to move the new ut to some other place